tests/cypress/tests/e2e/mlr/release.stepdef.js

```diff
@@ -24,7 +24,7 @@ When("I create, fill, and submit a report", () => {
   //Using the mlr.json as a guide, traverse all the routes/forms and fill it out dynamically
   traverseRoutes(template.routes);
 
-  cy.wait(2000);
+  cy.wait(5000);
```

Comment:
5000 is used in other areas of the release - Changed to keep it the same across the board.